autoPyTorch/evaluation/abstract_evaluator.py

```diff
@@ -861,7 +867,7 @@ def file_output(
             pipelines = None
 
         if hasattr(self, 'pipeline') and self.pipeline is not None:
-            if 'pipeline' not in self.disabled_file_outputs:
+            if 'pipeline' not in self.disable_file_output:
                 pipeline = self.pipeline
             else:
                 pipeline = None
```

Comment:
Actually, I am making such refactoring changes to the abstract evaluator in an upcoming PR to fix #349 . In this PR, my focus was to add a method to fit a pipeline. I changed `disable_file_output` because it was very messy.